app.js

```diff
@@ -10,6 +31,141 @@ function Book(title, author, pages, read) {
     }
 }
 
-myBook = new Book('Lord of the Rings', 'Tolkien', 365, true);
+function createBook(book) {
+  myLibrary.push(book);
+  document.querySelector('.input-form').classList.toggle('form-visibility');
+  localStorage.setItem("myLibrary", JSON.stringify(myLibrary));
+  render(myLibrary);
+}
+
+// Reads input values from the form.
+function readInputValues(){
+  return {
+  bookTitle: document.getElementById("book-title").value,
+  authorName: document.getElementById("author-name").value,
+  description: document.getElementById("book-description").value,
+  pagesNum: document.getElementById("book-pages").value,
+  status: document.querySelector('input[name="read"]:checked').value
+  }
+}
+function addBookToLibrary() {
+  let bookInfo = readInputValues();
+  const id = randomId();
+  let book =new Book(id, bookInfo.bookTitle, bookInfo.authorName, bookInfo.description, bookInfo.pagesNum, bookInfo.status);
+  let valid = inputValid(bookInfo.bookTitle, bookInfo.authorName, bookInfo.description, bookInfo.pagesNum);
+
+  if (valid == true) {createBook(book);}
+  clearForm();
+}
```

Comment:
**[OPTIONAL]** Let's use destructuring to get the values from the `readInputValues` function. Something like this `let {bookTitle, authorName, description, pagesNum, status} = readInputValues()` that way you can use the name of the properties as variables, preventing you from having to write `bookInfo.bookTitle` and just invoke it with `bookTitle`